portfolio/src/main/webapp/index.html

```diff
@@ -1,139 +1,168 @@
 <!DOCTYPE html>
```

Comment:
The changes to this file look like a lot, but it is just better formatting according to the Google style guide.